src/modules/Message/components/TextFragment/index.tsx

```diff
@@ -19,6 +20,8 @@ export default function TextFragment({
   tokens,
 }: TextFragmentProps): React.ReactElement {
   const messageStore = useMessageContext();
+  // fork note: custom render property
+  const { renderMessageMentionLabel } = useGroupChannelContext();
```

Comment:
unfortunately this needs to be prop drilled all the way down to avoid `useGroupChannelContext` causing it to rerender on every message update (regardless if the message changed or not)

Reply:
Oh ok! Refactored to use prop drilling instead.